backend/homework/7/src/main/java/sanskar/Main.java

```diff
@@ -0,0 +1,66 @@
+package sanskar;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import sanskar.config.AppConfig;
+import sanskar.logging.Logging;
+import sanskar.model.Vehicle;
+import sanskar.service.VehicleServiceFactory1;
+import sanskar.service.VehicleServiceFactory2;
+
+public class Main {
+
+    @Autowired
+    @Qualifier("vehicleServiceFactory1")
+    private VehicleServiceFactory1 vehicleFactory1;
+
+    @Autowired
+    @Qualifier("vehicleServiceFactory2")
+    private VehicleServiceFactory2 vehicleFactory2;
+
+    private static Logging logger = new Logging();
+
+    private static final String TYRE_LABEL = ", Tyre: ";
+    private static final String SPEAKER_LABEL = ", Speaker: ";
+    private static final String PRICE_LABEL = "Price: ";
+
+    public static void main(String[] args) {
+        var context = new AnnotationConfigApplicationContext(AppConfig.class);
+        Main main = context.getBean(Main.class);
+        main.generateAndPrintVehicleDetails();
+    }
+
+    void generateAndPrintVehicleDetails() {
+        vehicleFactory1.generateVehicles();
```

Comment:
Use meaningful and descriptive names for your variables rather than using numerical suffixes.